src/Core/Domain/Currency/Exception/CannotRemoveDefaultCurrencyFromShopAssociationException.php

```diff
@@ -26,9 +26,51 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Currency\Exception;
 
+use Throwable;
```

Comment:
We have to support php5.6 so we cannot use `Throwable`

Reply:
evil phpStorm ide auto included this for me :imp: I'll fix that :+1:

Reply:
Come to Emacs my friend :trollface: Come to the dark side!